QualityControl/public/layout/view/panels/settings.js

```diff
@@ -31,6 +31,7 @@ export default (model) =>
       (e)=> model.layout.setLayoutProperty('description', e.target.value),
     ),
     displayObjectTime(model),
+    applyFilters(model.layout),
```

Comment:
I would recommend this to be last so that is as close as possible to the sideTree. 
I have to admit I did not see the option when testing and I only found out about it during codee review